x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -217,22 +244,26 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
 
     const { getRecoveredAlerts } = services.alertFactory.done();
     const recoveredAlerts = getRecoveredAlerts();
+
     for (const alert of recoveredAlerts) {
-      const recoveredAlertId = alert.getId();
-      const indexedStartedDate = getAlertStartedDate(recoveredAlertId) ?? startedAt.toISOString();
-      const viewInAppUrl = getViewInAppUrlInventory(
-        criteria,
-        nodeType,
-        indexedStartedDate,
-        libs.basePath
-      );
+      const recoveredAlertInstanceId = alert.getId();
+      const indexedStartedDate =
+        getAlertStartedDate(recoveredAlertInstanceId) ?? startedAt.toISOString();
+      const alertUuid = getAlertUuid(recoveredAlertInstanceId);
+
       const context = {
-        group: recoveredAlertId,
+        alertDetailsUrl: getAlertDetailsUrl(libs.basePath, spaceId, alertUuid),
         alertState: stateToAlertMessage[AlertStates.OK],
-        timestamp: startedAt.toISOString(),
-        viewInAppUrl,
-        threshold: mapToConditionsLookup(criteria, (c) => c.threshold),
+        group: recoveredAlertInstanceId,
         metric: mapToConditionsLookup(criteria, (c) => c.metric),
+        threshold: mapToConditionsLookup(criteria, (c) => c.threshold),
+        timestamp: startedAt.toISOString(),
+        viewInAppUrl: getViewInInventoryAppUrl(
+          criteria,
+          nodeType,
+          indexedStartedDate,
+          libs.basePath
+        ),
```

Comment:
In an earlier comment you mentioned

> While I was there I also updated the `context.viewInApp` parameter to be space aware as well.

Did you intend to apply this to the inventory threshold alert too?

Reply:
@weltenwort Correct. Will update.